infrastructure/containers/postgres/automatic-backup/backup_or_restore.py

```diff
@@ -131,10 +187,21 @@ def restore_database(backup_fpath):
     )
     run_shell_command(create_db_cmd, env_vars=pg_env_vars)
 
+    # find the most recent backup or verify the specify backup exists
+    print("Getting backup file from S3")
+    session = create_s3_session()
+    if backup_obj_name is None:
+        backup_obj_name = get_most_recent_backup_obj_name(session)
+    elif backup_obj_name not in list_bucket_objects(session, BACKUP_BUCKET):
+        return False
+
+    # download the backup
+    download_backup_object(session, BACKUP_BUCKET, backup_obj_name, backup_obj_name)
```

Comment:
When you call functions, can you name the parameters you're passing in? `session=session, bucket=BACKUP_BUCKET, etc.`. I mean, you're variable names are super clear, it is just a good practice and often makes readability much quicker. It also protects this function call from breaking if we ever switch up the order of the arguments in the function definition.

Reply:
still not sure on the balance as in do we do it for every function or just ones with more than 1 parameter? or whats the rule here?

Reply:
I added them for all function calls with more than 1 parameter.